src/variables.jl

```diff
@@ -925,6 +925,44 @@ function _moi_add_constrained_variables(
     return var_indices
 end
 
+"""
+    reduced_cost(x::AbstractVariableRef)::Float64
+
+Return the reduced cost associated with variable `x`.
+
+Equivalent to querying the shadow price of the active variable bound
+(if one exists and is active).
+
+See also: [`shadow_price`](@ref).
+"""
+function reduced_cost(x::AbstractVariableRef)::Float64
+    model = owner_model(x)
+    if !has_duals(model)
+        error("Unable to query reduced cost of variable because model does" *
+              " not have duals available.")
+    end
+    # I am not sure if a feasibility sense model may have duals, but the
+    # sense may be changed after solving and having an error message help
+    # to discover this mistake.
+    obj_sense = objective_sense(model)
+    if obj_sense != MOI.MIN_SENSE && obj_sense != MOI.MAX_SENSE
+        error("The reduced cost is not available because the objective" *
+              " sense $obj_sense is not minimization or maximization.")
+    end
+    sign = objective_sense(model) == MOI.MIN_SENSE ? 1.0 : -1.0
```

Comment:
If, by "FEASIBILITY", we mean `min 0' x`, then sure, feasibility problems can have duals. I would just say that 
`sign = objective_sense(model) == MOI.MAX_SENSE ? -1.0 : 1.0` to cover both min and feasibility.

Reply:
I was not sure this is the meaning of FEASIBILITY_SENSE and for this reason I included the error. I removed it in the last commit, as it is not of my interest I will not dispute this.

Reply:
I agree with odow, using `-1`on `MAX_SENSE` only seems more sensible